src/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-Do-List</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+</head>
+<body>
+    <section class="box-list">
+    <div class="list-container">
+        <div class="list-title">
+            <h3>Todays To Do</h3>
```

Comment:
**[OPTIONAL]**
_- If you define an h3, to maintain best practices on HTML, I suggest to you to start with an h1, or use span instead._